os/macosx/openttd.entitlements

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.cs.allow-dyld-environment-variables</key>
```

Comment:
This lacks information why this entry is here (as for the next), so it is easily forgotten in any refactor. I would suggest to add an XML comment here, referencing the Steamwork documentation for example.